### DISCUSSION TITLE
Text field stretching occasionally

### DISCUSSION THREAD

**Discussion by yaroslavyaroslav** [community/community#38460] [Mobile]
I've failed to manage this one from the beginning, so just wanted to made it be counted here. 

https://github.com/orgs/community/discussions/35578

The issue is still appears on all iOS devices. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.87.0 (82263682)
Device: iPhone 12 mini (iOS 16.1)
</details>

**Comment by aconsuegra** [comment]
Hi @yaroslavyaroslav. Thanks for getting in touch! We are aware of the issue and we are working on a fix that we hope to have ready soon.

**Comment by yaroslavyaroslav** [comment]
It worked well, I guess I'd added it somewhere in comments in the past.
